codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/FluentClientDecorator.kt

```diff
@@ -563,6 +562,30 @@ class FluentClientGenerator(
     }
 }
 
+fun generateOperationShapeDocs(writer: RustWriter, symbolProvider: SymbolProvider, operation: OperationShape, model: Model): List<String> {
+    val input = operation.inputShape(model)
+    val operationName = symbolProvider.toSymbol(operation).name
+    val fluentBuilderFullyQualifiedName = "crate::client::fluent_builders::$operationName"
+    return input.members().map { member ->
+        val memberName = symbolProvider.toMemberName(member)
+        val memberSymbol = symbolProvider.toSymbol(member)
+        val outerType = memberSymbol.rustType()
+
+        val builderInput = "$memberName(${outerType.stripOuter<RustType.Option>().render(fullyQualified = false)})"
+        val builderInputLink = "$fluentBuilderFullyQualifiedName::$memberName"
+        val builderSetter = "${member.setterName()}(${outerType.render(fullyQualified = false)})"
+        val builderSetterLink = "$fluentBuilderFullyQualifiedName::${member.setterName()}"
```

Comment:
I think there's a lot of potential for this to break, for instance, if we modify the codegen for the actual functions but forget that these docs exist also. Consider pulling out some helper functions for this in the function codegen, and then calling those same functions here to get these.

Reply:
@jdisanti I updated things a bit, can you let me know if you like it more?